sysvabi64/sysvabi64.rst

```diff
@@ -916,10 +923,652 @@ These follow the AAELF64_ base definition from Arm.
+
+ * ``.got.plt`` for code addresses accessed only from the Procedure Linkage
+   Table (PLT).
+
+ * ``.got`` all other addresses and offsets.
+
+Both the ``.got`` and ``.got.plt`` sections are aligned to a minimum
+of 8 bytes. They both contain 8-byte sized entries.
+
+If a position-independent program requires direct access to the
+absolute address of a symbol, that symbol will have a GOT
+entry. Because the executable and shared objects have their own
+independent GOTs, a symbol's address may appear in several tables. The
+dynamic linker processes all of the relocations in the ``.got``
+section before giving control to any code in the process image, thus
+ensuring the absolute addresses are available during execution. Code
+in the process that runs as part of a GNU indirect function resolver
+is an exception to this rule. See `GNU Indirect Functions`_ for the
+restrictions on what addresses can be accessed from a resolver
+function.
+
+By convention the ``.got`` section's first entry (number zero) is
+reserved to hold the address of the dynamic structure, referenced with
+the linker defined symbol ``_DYNAMIC``. Versions of glibc prior to
+version 2.35 use this address to find its own dynamic structure
+without without having yet processed their own relocation entries.
+
+AArch64 entries one and two in the ``.got.plt`` are reserved for the
+dynamic linker.
+
+For AArch64 the linker defined ``_GLOBAL_OFFSET_TABLE_`` symbol should
+be the address of the first global offset table entry in the ``.got``
+section.
+
+See `Sample code sequences for code models`_ for code model for
+examples of how to access the ``.got``.
+
+Function Addresses
+^^^^^^^^^^^^^^^^^^
+
+Direct function calls are those where the name of the called function
+is known at compile time. The PC-relative direct branch instructions
+may be used for all direct function calls, whether absolute or
+position-independent.
+
+Indirect function calls are those where the address of the function is
+in a pointer. Appropriate code is used to load the value of the
+pointer into a register, as for other data, and then an indirect
+branch instruction is used with the register as an operand to the
+instruction.
+
+In statically linked code the address of a function is always its real
+address. However in dynamically linked environments references to the
+address of a function from an executable file and its shared objects
+might not resolve to the same value.
+
+References from within shared objects to a function address will
+normally be resolved by the dynamic linker to the virtual address of
+the function itself.
+
+References from within a non-position-independent executable file to a
+function address defined in a shared object will normally be resolved
+by the static linker to the address of the `Procedure Linkage Table`_
+(PLT) entry for that function within the executable file. If the
+address is in a writeable location the linker may use a dynamic
```

Comment:
s/writeable/writable/

Reply:
Ack. I think writeable is an alternative spelling in British English, but writable is more appropriate here, I've also changed the other occurrences in the doc.